tests/baselines/reference/ts-expect-error.errors.txt

```diff
@@ -67,4 +67,18 @@ ts-expect-error.ts(40,2): error TS2367: This comparison appears to be unintentio
 !!! error TS2367: This comparison appears to be unintentional because the types 'true' and 'false' have no overlap.
     (({ a: true } as const).a === false); // error
      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS2367: This comparison appears to be unintentional because the types 'true' and 'false' have no overlap.
\ No newline at end of file
+!!! error TS2367: This comparison appears to be unintentional because the types 'true' and 'false' have no overlap.
+    
+    // @ts-expect-error: additional commenting with no whitespace
+    var invalidCommentedFancySingle: number = 'nope';
+    
+    /*
+     @ts-expect-error: additional commenting with no whitespace */
+    var invalidCommentedFancyMulti: number = 'nope';
+    
+    // @ts-expect-error--additional commenting with no whitespace

```

Comment:
it works because `@ts-expect-error` and `@ts-ignore` use a totally different regex: https://github.com/microsoft/TypeScript/blob/e374eba/src/compiler/scanner.ts#L335-L340

for these two pragmas you can even write strange stuff like `@ts-expect-errors` or `@ts-ignored` ([playground link](https://www.typescriptlang.org/play?#code/FAehAIAEBcGcFoCmAPADogxtJAnHB7HASwDsBzYARnAGpwBvAX2FAhgSLJMMQBN4A7gEMcJUhWp0mwIA))

(but as per your suggestions below, I've removed this test case)